### DISCUSSION TITLE
issue - "Save does not match client version" since second log in

### DISCUSSION THREAD

**Discussion by bahraoui** [daye10/dokkan-battle-farm-bot-app#12] [General]
Hi, i have an issue anytime I want to login from save. If it's the first time and I load my save then I constantly have "Save does not match client version". This means I have to reconnect via Facebook each time to log in.

---

Salut, j'ai un problème quand j'essaie de me log pour la 2ème fois. À chaque fois que je me connecte pour la première fois, tout se passe correctement, mais dès que je quitte pour revenir plus tard, j'ai le message "Save does not match client version". Si le problème ne vient pas de ma machine, je suppose qu'il ya un problème de version quelque part.
Aussi, petit point pas dérangeant : des warnings python s'affichent quand on choisit une save pour le daily login.

**Comment by daye10** [comment]
Salut ! 
Pour la prochaine fois le mieux serait de signler le probleme ici : https://github.com/daye10/dokkan-battle-farm-bot-app/issues

Pour faire simple, lorsque tu saves un compte sur lequel tu t'es connecte depuis FB, le format de ton identifiant de connexion est different, ce qui fait que le bot a du mal a lire le fichier, pour regler ce probleme c'est tres simple.

Tu peux ouvrir le fichier avec n'importe quel editeur de texte (Bloc-notes, word, ...), tu devrais avoir quelque chose comme ceci : 

azd1TUFvN0dHakc4TFY5L3MyWk1xaFB1TEpLRnRtOXowYjVURXF2dUFIU1RO
RnZRK0o3cEh2Vit3cU1zSFZqWE1EUkl6ZnlmYUwvOXJZUEJmVW53enc9PTo1
dmIyYlZ1OUt3bjRvYWRvdEkxbVRRPT0=

6633ae74-eee4-4cd8-bc46-fe062c0d4131
6bfe07a7-73ef-4d11-9886-bd85b31dbe52:94ca3314
android
global


Tout ce que tu as a faire, c'est de concatener les 3 premieres (les mettres les une apres les autres pour avoir ue seule et unique ligne) lignes, et retirer la ligne vide, comme ceci :


azd1TUFvN0dHakc4TFY5L3MyWk1xaFB1TEpLRnRtOXowYjVURXF2dUFIU1RORnZRK0o3cEh2Vit3cU1zSFZqWE1EUkl6ZnlmYUwvOXJZUEJmVW53enc9PTo1dmIyYlZ1OUt3bjRvYWRvdEkxbVRRPT0=
6633ae74-eee4-4cd8-bc46-fe062c0d4131
6bfe07a7-73ef-4d11-9886-bd85b31dbe52:94ca3314
android
global

**Reply by bahraoui** [reply]
Désolé, la prochaine fois (s'il yen a une) je le signalerais au bon endroit.
Merci beaucoup.
